src/Codata/Thunk.agda

```diff
@@ -8,12 +8,27 @@
 
 module Codata.Thunk where
 
+open import Level
 open import Size
-open import Relation.Unary
+
+private
+  variable
+    ℓ ℓ₁ ℓ₂ : Level
 
 ------------------------------------------------------------------------
 -- Basic types.
 
+SizedType : (ℓ : Level) → Set (suc ℓ)
+SizedType ℓ = Size → Set ℓ
+
+∀ˢ[_] : SizedType ℓ → Set ℓ
```

Comment:
Can't we give these the same name as the `Relation.Unary` counterparts?
If people want to use both in the same file they can rename them / use
qualified names at the point of use.

Reply:
Ok, but I found that these are only used in `Codata.Thunk`.  In other modules, they would clash, e.g.
```
  Checking Codata.M (Codata/M.agda).
Codata/M.agda:31,18-19
Ambiguous name _⇒_. It could refer to any one of
  Size._⇒_ bound at
    Size.agda:33,1-4
  C._⇒_ bound at
    Data/Container/Core.agda:41,8-11
```
I think it is not good that these are in scope when just `open import Size`.  This module is known to be slim and to not generate conflicts.
As a compromise, I put these definition into a submodule `SizedType`.

Reply:
Hmm I think a better solution is to mimic the normal module hierarchy underneath `Size` just as we do elsewhere in the library. For example we would put `∀[_]` in `Size.Relation.Unary` and `_⇒_` in `Size.Function`. That would make it so much easier to expand these later on if we found it necessary.

Btw, I haven't seen `⇒` used to denote functions very much. Did you consider using `→` instead which would be significantly less likely to clash with other parts of the library?

Reply:
Another solution would be to strip the uses of `∀[_]` and `_⇒_` from `Codata.Thunk` by expanding to ordinary uses of `∀` and `->`.  It is just a few characters more. Then we do not have to discuss this problem further.  

> put `∀[_]` in `Size.Relation.Unary` and `_⇒_` in `Size.Function`

We should maybe not have two top-level modules containing one definition each...  (That are then used in a single other module, `Codata.Thunk`).